patches/react-native+0.72.14.patch

```diff
@@ -1,55 +0,0 @@
-diff --git a/node_modules/react-native/React/CoreModules/RCTTiming.mm b/node_modules/react-native/React/CoreModules/RCTTiming.mm
```

Comment:
We need this patch on 0.73 too. It might require some adjustments.

Reply:
I though this was fixed after 0.73.4.

Reply:
@saghul I added back the patch file.

Reply:
Last I checked it landed on 0.74

Reply:
That comes after 0.73.8. I'll try to update.

Reply:
Don't! It's a bigger update.

Reply:
Let's stick to 0.73 as planned, and once 0,.75 is out we can update to 0.74.latest.